docs/asynchronously-loading-translations.md

```diff
@@ -38,9 +38,9 @@ export default Ember.Route.extend({
   intl: Ember.inject.service(),
   
   async beforeModel() {
-    const translations = await fetch('/translations/en-us.json');
-    
-    this.get('intl').addTranslations('en-us', translations.json());
+    const response = await fetch('/translations/en-us.json');
+    const translations = await response.json();
+    this.get('intl').addTranslations('en-us', translations);
```

Comment:
I think you can also change it to

```js
const translations = await fetch('/translations/en-us.json');
    
this.get('intl').addTranslations('en-us', await translations.json());
```

which might be cleaner

Reply:
@makepanic Done as per comment.